pages/_document.js

```diff
@@ -5,11 +5,61 @@ import sprite from 'svg-sprite-loader/runtime/sprite.build';
 
 import { mediaStyles } from 'gfw-components';
 
-const isProduction = process.env.NODE_ENV === 'production';
+const isProduction = process.env.NEXT_PUBLIC_FEATURE_ENV === 'production';
```

Comment:
found out, this is the correct env var we are actually using to discriminate between environments, not `NODE_ENV`